examples/graphene_apps/cppopenvino/test/test_send_request.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python3
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import os
+import subprocess
+import zmq
+import json
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+# -------------------------------------------------------------------------
+
+
+# This is a test file to send work order.
+def main(argv):
+    if len(argv) >= 1:
+        workload = argv[0]
+    else:
+        workload = None
+
+    if len(argv) >= 2:
+        params = argv[1]
+    else:
+        params = None
+
+    context = zmq.Context()
+    # TODO: read zmq url from config file.
+    zmq_url = "tcp://ov-work-order:8888"
+    req = {}
+    req["workloadId"] = workload
+    req["params"] = params
+    logger.info("Send work order request: {}".format(json.dumps(req)))
+    _send_request_zmq(context, zmq_url, json.dumps(req))
+
+# -------------------------------------------------------------------------
+
+
+def _send_request_zmq(context, zmq_url, workload):
+    """
+    Send workload via ZMQ and returns the response string.
+    Parameters :
+        zmq_url: zmq url to send workload.
```

Comment:
Add @param, for doc string.

Reply:
My understanding is we need @param doxygen only for C++ and it's not needed for python.